src/cmr/sizing/formats.clj

```diff
@@ -29,20 +35,31 @@
       (get-in [:umm :SizeEstimation :AverageSizeOfGranulesSampled])
       read-number))
 
-(defn- get-avg-compression-rate-ascii
-  "Gets SizeEstimation value for AvgCompressionRateASCII and parses it to a number."
-  [variable]
-  (-> variable
-      (get-in [:umm :SizeEstimation :AvgCompressionRateASCII])
-      read-number))
-
 (defn- get-avg-compression-rate-netcdf4
   "Gets SizeEstimation value for AvgCompressionRateNetCDF4 and parses it to a number."
   [variable]
   (-> variable
       (get-in [:umm :SizeEstimation :AvgCompressionRateNetCDF4])
       read-number))
 
+(defn- get-avg-fill-value-digit-number
+  "Get the average digit number for each value of FillValues in the variable.
+  For example, FillValues containing [{:Value 1} {:Value 100}], returns (1+3)/2=2."
+  [variable]
+  (let [fvs (get-in variable [:umm :FillValues])
+        fvs-digit-numbers (map #(count (str (:Value %))) fvs)]
```

Comment:
If there are no fill values in the variable metadata you will end up dividing by zero. Are FillValues a required field in UMM-Var?

If they are optional we might need to just pick an expected fill value of say 3 digits.

Reply:
I just double-checked - both Fill Values and ValidRanges are optional. We're going to have to figure out what to do when they are not provided.

https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/variable/v1.3/umm-var-json-schema.json
```
      "ValidRanges": {
        "description": "Valid ranges of variable data values.",
        "type": "array",
        "items": {
          "$ref": "#/definitions/ValidRangeType"
        },
        "minItems": 0
      },
```

```
      "FillValues": {
        "description": "The fill value of the variable in the data file. It is generally a value which falls outside the valid range. For example, if the valid range is '0, 360', the fill value may be '-1'. The fill value type is data provider-defined. For example, 'Out of Valid Range'.",
        "type": "array",
        "items": {
          "$ref": "#/definitions/FillValueType"
        },
        "minItems": 0
      }
```

Reply:
I thought about divided by 0 case, but since I didn't know what to use in that case, I just left it there to get your feedbacks :)